### PR TITLE
Add back network page

### LINKED ISSUE #524: Finish Network Page
Few remaining items to get the network page completed.

On /network:
1)  - [x] Update this information on /network (check with protocol team/comms for dates)
image
2)  - [x] Any "node operator callout" should be the following: "Node Operators" -> Body: "View the current nodes in the network and who is running them. You can use Flow Port to delegate or stake to a node. The picture should be the one of the 'abstract images' used across the site (use best discretion)

When clicking into a details page of a network (/network/:name):
3) - [x] We need to update 
image

The RSS feed should point to the RSS subscription feed available at status.onflow.org (in the subscribe for updates). 

Upcoming spork date should be filled in as per protocol team.

4) - [x] For past spork information, they need to be parsed in through this json file: https://github.com/onflow/flow/blob/master/sporks.json

5) - [x] Once 4 is done, the navigation under the network tab should change the link of upcoming and past spork information to those pages. We should also remove or redirect the internal pages in /nodes that contain this same information.

### PR DESCRIPTION
**Pull request by aureentuluvacal**
Closes #524

### CODE REVIEW

**Comment by aureentuluvacal** [comment]
@srinjoyc @10thfloor Is there any data I can pull for Canarynet? The sporks info only includes mainnet and testnet. Also, for previous spork dates, would you like that to be dynamic?

File: app/ui/design-system/src/lib/Components/InternalContentLink/index.tsx [hunk 1]
```diff
@@ -29,7 +29,7 @@ export function InternalContentLink({
     return (
       <AppLink
         to={href}
-        className={clsx(defaultClasses, {
+        className={clsx(defaultClasses, className, {
```
Comment (aureentuluvacal) [inline app/ui/design-system/src/lib/Components/InternalContentLink/index.tsx:32]: Noticed we weren't actually using className in the component 😂

File: app/ui/design-system/src/lib/Components/SporksCard/index.tsx [hunk 2]
```diff
@@ -1,10 +1,10 @@
+import { Disclosure } from "@headlessui/react"
 import clsx from "clsx"
-import { format, parseISO } from "date-fns"
-import { useState } from "react"
 import { ReactComponent as CopyIcon } from "../../../../images/action/copy"
 import { ReactComponent as ChevronDownIcon } from "../../../../images/arrows/chevron-down"
 import { ReactComponent as ChevronUpIcon } from "../../../../images/arrows/chevron-up"
 import { SporkMetadata } from "../../interfaces"
+import { dateYYMMDD } from "../../utils/dates"
```
Comment (aureentuluvacal) [inline app/ui/design-system/src/lib/Components/SporksCard/index.tsx:7]: For consistent dates

File: app/routes/network/data.ts [hunk 3]
```diff
@@ -1,11 +1,11 @@
 import { Article } from "~/ui/design-system/src/lib/interfaces"
+import DefaultImage from "~/ui/design-system/images/misc/Asset_8@3x.png"
 
 export const featuredArticle: Article = {
-  heading: "Node operator callout",
+  heading: "Node operators",
```
Comment (10thfloor) [inline app/routes/network/data.ts:5]:
```suggestion
  heading: "Node Operators",
```

**Review by 10thfloor — APPROVED** [review]

**Review by 10thfloor — APPROVED** [review]

**Pull request merged**